api/src/Service/RestHandler/OrganisationRestHandler.php

```diff
@@ -95,7 +97,28 @@ public function create(array $data): Organisation
     }
 
     /**
-     * @param array $data
+     * @param int $id
+     * @return bool
+     * @throws OptimisticLockExceptionAlias
+     * @throws \Doctrine\ORM\ORMException
+     */
+    public function delete(int $id)
+    {
+        if (null === ($organisation = $this->orgRepository->find($id))) {
+            throw new \Doctrine\ORM\ORMException();
```

Comment:
Not sure if this is the right exception to be throwing.

Reply:
From looking at the class it seems to be intended for use when something has gone wrong within Doctrine rather than if doctrine hasn't been able to find an entity. Unless we're going to handle the exception based on the exception type thrown then a regular Exception will probably be fine here.

If you do decide to go down a custom exception route then see the `/Exception` folder for how others have been structured.